tests/bot/exts/filters/test_antimalware.py

```diff
@@ -105,20 +105,35 @@ async def test_python_file_redirect_embed_description(self):
 
     async def test_txt_file_redirect_embed_description(self):
         """A message containing a .txt file should result in the correct embed."""
```

Comment:
This should be changed to include the other text like files

Reply:
Resolved in aeaef8ff604c9ea62fdf1602200ee87f2adf7f6a.